packages/anon-aadhaar-react/src/components/ProveButton.tsx

```diff
@@ -20,7 +21,7 @@ export const ProveButton: React.FC<ProveButtonProps> = ({
   provingEnabled,
   setErrorMessage,
 }) => {
-  const { state, startReq, appId } = useContext(AnonAadhaarContext)
+  const { state, startReq, appId, testing } = useContext(AnonAadhaarContext)
```

Comment:
I think testing  should be env variable.

Reply:
Not sure, let's say you want to push to production but still want to use test pdf.

Reply:
It not make sense, because people still want test with mock pdf and run on real data as same time.